
uint64 ALL_FAULTS = 0

std_msgs/Header header
uint64 fault_id
